src/components/plots/PlotHeader.jsx

```diff
@@ -154,37 +131,28 @@ const Header = (props) => {
     setResetDisabled(true);
   };
   return (
-    <Row>
-      <Col span={16}>
-        <PageHeader
-          style={{ width: '100%', paddingTop: '12px', paddingBottom: '6px' }}
-          title='Edit collection'
-          breadcrumb={{ routes: baseRoutes, itemRender }}
-          subTitle={`Last saved: ${saveString}`}
-          extra={(
-            <Space>
-              <FeedbackButton />
-              <ReferralButton />
-              <Button
-                key='reset'
-                type='primary'
-                onClick={onClickReset}
-                disabled={resetDisabled}
-              >
-                Reset
-              </Button>
-            </Space>
-          )}
-        />
-      </Col>
-    </Row>
+    <Header
+      title={title}
+      extra={(
+        <Space>
```

Comment:
There is no need to use `Space` here if it only contains 1 child component.

Reply:
I removed it